printer_zpl2/models/printing_label_zpl2.py

```diff
@@ -69,7 +69,7 @@ class PrintingLabelZpl2(models.Model):
     test_print_mode = fields.Boolean(string="Mode Print")
     test_labelary_mode = fields.Boolean(string="Mode Labelary")
     record_id = fields.Integer(string="Record ID", default=1)
-    extra = fields.Text(string="Extra", default="{}")
```

Comment:
That is to please `pre-commit run -a`. Because Odoo auto-generates string from field name (replace `_` by space then change first letters of words to uppercase) and the generated one is not different than the string so there was warning with it..

Reply:
ok got it, thanks, pylint_odoo `attribute-string-redundant`